pages/progress-list/index.js

```diff
@@ -35,38 +44,148 @@ export default function ProgressList({ moods, setMoods }) {
   };
 
   if (!moods.length) {
-    return <h2>No Content yet...</h2>;
+    return (
+      <>
+        <h2>Noch kein Inhalt!</h2>
+        <p>
+          Hier werden deine Stimmungen gespeichert. Diese kannst du eingeben,
+          indem du einen Flow beendest und auf &quot;Fertig&quot; klickst.
+        </p>
+      </>
+    );
   }
 
   if (!Array.isArray(moods)) {
-    return <h2>No Content</h2>;
+    return <h2>Kein Inhalt</h2>;
   }
 
   return (
     <>
-      <h2>Meine Stimmungen:</h2>
+      <StyledHeading>Meine Stimmungen:</StyledHeading>
       <ul>
-        {moods.map((entry, index) => (
-          <li key={index}>
-            <p>{entry.date}:</p>
-            {editedIndex === index ? (
-              <EditForm
-                entry={entry}
-                onSubmit={(updatedEntry) => handleSave(index, updatedEntry)}
-              />
-            ) : (
-              <>
-                Stimmung: {entry.mood}, Körperliches Wohlbefinden: {entry.body},
-                Ich habe {entry.duration} Minuten Yoga gemacht.
-                {entry.comment && <>, Kommentar:{entry.comment}</>}
-                <button onClick={() => handleEdit(index)}>✏️</button>
-                <button onClick={() => handleDelete(index)}>🗑️</button>
-              </>
-            )}
-          </li>
-        ))}
+        {moods
+          .slice(0)
+          .reverse()
+          .map((entry, index) => (
+            <StyledListItem key={index} lastChild={index === moods.length - 1}>
```

Comment:
It's not recommended to use the index as a key in dynamic rendering. If your entries don't have ids you could generate them with `uid`, `uuid` or `nanoid` whenever an entry is created.

Reply:
Oh thank you I'll try to fix that.